JSDemos/Demos/Form/ColumnsAdaptability/description.md

```diff
@@ -1 +1,21 @@
-The Form component can use different layouts depending on&nbsp;the screen width. The dependency between column count and the screen width is&nbsp;specified using the **colCountByScreen** property, which has a&nbsp;higher priority than the **colCount** and the **minColWidth** properties. In&nbsp;this demo, the form is&nbsp;displayed using a&nbsp;4-column layout on&nbsp;wide screens and a&nbsp;2-column layout on&nbsp;narrow screens. Switch between the landscape and portrait layouts to&nbsp;see how the form looks on&nbsp;wide and narrow screens.
\ No newline at end of file
+The Form component can use different layouts depending on the screen width. Click one of the buttons on the right from the demo title to switch between horizontal and vertical orientations. The number of columns in the layout can be specified in one of the following manners:
```

Comment:
Avoid passive voice whenever possible: http://documentation.internalpages.devexpress.devx/documentation-guidelines/language-guidelines/passive-voice/
```suggestion
The Form component can use different layouts depending on the screen width. Click one of the buttons on the right from the demo title to switch between horizontal and vertical orientations. You can determine the number of columns on the Form in one of the following manners:
```

Reply:
I have a question about the preposition '**on** the Form'. It looks a bit weird and we usually say 'in the Form/component'. Can we discuss it?

Reply:
We can. If you treat the Form as a surface where you can place something, you say "on" (like I always do). If you consider the Form more like a container, that would be more correct to say "in". This attitude can be platform-specific, and maybe everyone have used to say "in" and treat forms as containers.

Reply:
If you say "in the Form" everywhere, just continue to do so. This is correct.

Reply:
I would like to keep it consistent and make the following changes: `You can determine the number of columns in the Form in one of the following manners:`

Reply:
If this is consistent with the rest of your documentation, I have no objections.
```suggestion
The Form component can use different layouts depending on the screen width. Click one of the buttons on the right from the demo title to switch between horizontal and vertical orientations. You can determine the number of columns in the Form in one of the following manners:
```